tests/e2e/README.md

```diff
@@ -75,12 +75,29 @@ kubectl create clusterrolebinding myname-cluster-admin-binding    --clusterrole=
 
 ## Step 4: Export test script variables
 
-**Option 1:** Build your own images.
+**Option 1:** Build your own images and use an in-cluster registry.
```

Comment:
I wonder if we should make this default for the local VM cases in https://github.com/istio/istio/pull/5838#issuecomment-392213223